Sources/Core/UploadClient/UploadClient+Live.swift

```diff
@@ -1,6 +1,10 @@
 import Dispatch
 import Foundation
 
+#if canImport(FoundationNetworking)
+import FoundationNetworking
+#endif
```

Comment:
I assume the import is only successful for LInux and the conditonal check is for Mac? Or is it the other way around?

Reply:
I don't really know, but: this three-liner was previously in the various files that mention `URLResponse`, and apart from changing that to `HTTPURLResponse`, I've also added a reference to this file which didn't have one before…  without this conditional import the build broke in Linux but not macOS… so yeah I guess maybe that import is implicit on macOS but needs to be explicit on Linux, or something?